scripts/zones/Batallia_Downs/mobs/Skirling_Liger.lua

```diff
@@ -0,0 +1,49 @@
+-----------------------------------
+-- Area: Batallia Downs (105)
+--  MOB: Skirling Liger
+-----------------------------------
+
+require("scripts/globals/status");
+
+-----------------------------------
+-- onMobInitialize Action
+-----------------------------------
+
+function onMobInitialize(mob)
+end;
+
+-----------------------------------
+-- onMobSpawn
+-----------------------------------
+
+function onMobSpawn(mob)
+    if (mob:getMod(MOD_REGAIN) ~= 0) then
+        mob:setMod(MOD_REGAIN,0);
+    end
+end;
+
+-----------------------------------
+-- onMobFight
+-----------------------------------
+
+function onMobFight(mob, target)
+    mob:setMod(MOD_REGAIN, 50);
```

Comment:
I did something like what you have with Ghillie_Dhu.lua but that was because his regain was variable, not a fixed amount. If these also are variable, my previous suggestion will be wrong.